packages/smart-account/src/SmartAccount.ts

```diff
@@ -260,6 +260,9 @@ class SmartAccount extends EventEmitter {
         new ethers.providers.JsonRpcProvider(providerUrl),
         {
           dappAPIKey: clientConfig.dappAPIKey || '',
+          // Review: default false
+          // could come from global set config or method level when we implement fee mode
+          strictSponsorshipMode: false,
```

Comment:
what is fee mode here?
This is updated if user gives in constructor

Reply:
we can get it from options in constructor. I haven't been able to decide yet would it be at transaction level (sendTransaction unified method with type: and feeMode: ) or at smart account config level (fixed)

Reply:
I think we should take this in constructor, but will be better if we take in sendTransaction and override maybe